README.md

```diff
@@ -39,7 +39,7 @@ Embulk is a Java application. Please make sure that Java SE Runtime Environment
 You can download `embulk.bat` using this command on cmd.exe or PowerShell.exe:
 
 ```
-PowerShell -Command "& {Invoke-WebRequest http://dl.embulk.org/embulk-latest.jar -OutFile embulk.bat}"
+PowerShell -Command "& {[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::TLS12; Invoke-WebRequest http://dl.embulk.org/embulk-latest.jar -OutFile embulk.bat}"
```

Comment:
Wondered if we could have a description about the background before the command example, such as:

```
You can download `embulk.bat` using the command below on cmd.exe or PowerShell.exe.

[Bintray no longer supports TLS 1.1 since June, 2018](https://jfrog.com/knowledge-base/why-am-i-failing-to-work-with-jfrog-saas-service-with-tls-1-0-1-1/) although [PowerShell's `Invoke-WebRequest` uses only SSL 3.0 and TLS 1.1 by default](https://social.technet.microsoft.com/Forums/en-US/00b78ac4-cadb-4566-b175-beb9d34a9093/how-to-use-tls-11-or-12-for-invokewebrequest). You'll need a tweak for PowerShell to use TLS 1.2, such as:
```

Reply:
Thank you. I added your description as is.